src/vs/workbench/contrib/quickaccess/browser/commandsQuickAccess.ts

```diff
@@ -85,7 +85,7 @@ export class CommandsQuickAccessProvider extends AbstractEditorCommandsQuickAcce
 		super({
 			showAlias: !Language.isDefaultVariant(),
 			noResultsPick: (filter) => {
-				return this.interactiveSessionService.getProviderIds().length
+				return this.interactiveSessionService.getProviderInfos().length
 					? {
 						label: localize('askXInInteractiveSession', "Ask '{0}' in an Interactive Session", filter),
```

Comment:
Can you update this label with the id as well? Maybe like `Ask {0} '{1}'...` where 0 is the id and 1 is the filter.